src/elm/Page/PublicProfile.elm

```diff
@@ -0,0 +1,221 @@
+module Page.PublicProfile exposing (Model, Msg, init, jsAddressToMsg, msgToString, update, view)
+
+import Api.Graphql
+import Avatar
+import Eos
+import Eos.Account as Eos
+import Graphql.Http
+import Html exposing (..)
```

Comment:
Elm community tries to follow a few good practice patterns. One of them is only exposing what you'll need (https://github.com/ohanhi/elm-style-guide#module-definitions)

Just remove the `exposing (..)` and let the elm Compiler help you along the way